src/EpicsClient/EpicsClientMonitor.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+#include "EpicsClientFactory.h"
+#include "EpicsClientInterface.h"
+#include "Stream.h"
+#include <array>
+#include <atomic>
+#include <pv/pvAccess.h>
```

Comment:
EPICS headers are now exposed to users of `EpicsClient` which defeats the purpose of the PIMPL. We should avoid this if possible or otherwise remove the PIMPL.